test/test/lock/LockTests.java

```diff
@@ -27,12 +27,15 @@ public void datagramSocketLock(TestProcess p) throws Exception {
     public void raceToLocks(TestProcess p) throws Exception {
         int interval = Integer.parseInt(p.inputs()[0]);
         Output out = p.profile("--lock " + interval + " --threads -o collapsed");
-        Output stdout = p.readFile(TestProcess.STDOUT);
```

Comment:
If `stdout` is no longer read, then `output = true` attribute is not necessary?

Reply:
Not needed indeed, a38c091828e95da58709967b9d7b8e706e386436